lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -10441,15 +10441,25 @@ void Emit(ParseNode *pnode, ByteCodeGenerator *byteCodeGenerator, FuncInfo *func
         if (pnode->isUsed || fReturnValue)
         {
             byteCodeGenerator->StartStatement(pnode);
-            const Js::OpCode op = (pnode->nop == knopDecPost) ? Js::OpCode::Sub_A : Js::OpCode::Add_A;
+            bool isESBigIntEnabled = byteCodeGenerator->GetScriptContext()->GetConfig()->IsESBigIntEnabled();
```

Comment:
if BigInt is considered, we generate different bytecode for post increment/decrement. Instead of: convert to number and add 1 to it, we convert value to numeric and increment it.